src/main/scala/tech/cryptonomic/conseil/tezos/michelson/contracts/TokenLedgers.scala

```diff
@@ -0,0 +1,185 @@
+package tech.cryptonomic.conseil.tezos.michelson.contracts
+
+import tech.cryptonomic.conseil.tezos.TezosTypes.{AccountId, Contract, ContractId, Decimal, ScriptId}
+import tech.cryptonomic.conseil.tezos.TezosTypes.Micheline
+import tech.cryptonomic.conseil.tezos.michelson.dto.{
+  MichelsonBytesConstant,
+  MichelsonInstruction,
+  MichelsonIntConstant,
+  MichelsonSingleInstruction
+}
+import cats.implicits._
+import scala.util.Try
+import com.typesafe.scalalogging.LazyLogging
+
+/** Collects custom token contracts structures and operations */
+object TokenLedgers extends LazyLogging {
```

Comment:
`Ledgers` is a problematic word to use. Tezos itself is a ledger. I think it's better to call it `TokenContracts`.

Reply:
done